### PR TITLE
Migrated GitHubLabeler to v0.11

### PR DESCRIPTION
**Pull request by prathyusha12345**
## Summary
Migrated GItHubLabeler solution to v0.11 with the below changes.
1. Replaced data loading  by using LaodFromTextFile
2.Removed unnecessary namespace using statements.

### CODE REVIEW

File: machine-learning/tutorials/GitHubIssueClassification/GitHubIssueClassification.csproj [hunk 1]
```diff
@@ -15,7 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="0.10.0" />
+    <PackageReference Include="Microsoft.ML" Version="0.11.0-preview-27428-4" />
```
Comment (NextTurn) [inline machine-learning/tutorials/GitHubIssueClassification/GitHubIssueClassification.csproj]: Is preview version needed?

Comment (JRAlexander) [reply machine-learning/tutorials/GitHubIssueClassification/GitHubIssueClassification.csproj]: Hi @NextTurn! We are in process of updating this sample for version 0.11.0, and are using the preview version to make initial changes. After 0.11.0 release, we'll update this with the released version, make another review pass, update the tutorial, and push to live.

**Comment by prathyusha12345** [comment]
@JRAlexander Please merge the PR once you are fine the changes.

**Comment by JRAlexander** [comment]
Thanks, @prathyusha12345!

**Pull request merged**